wazimap_ng/profile/serializers/metrics_serializer.py

```diff
@@ -16,20 +16,20 @@ def get_data_for_key_metric_and_geographies(profile_key_metric, geographies):
         return indicator_data.first().data
     return None
 
-def absolute_value(profile_key_metric, geography):
+def absolute_value(profile_key_metric, geography, versions):
     indicator_data = get_data_for_key_metric_and_geographies(profile_key_metric, [geography])
     if indicator_data != None:
         return MetricCalculator.absolute_value(indicator_data, profile_key_metric, geography)
     return None
 
-def subindicator(profile_key_metric, geography):
+def subindicator(profile_key_metric, geography, versions):
     indicator_data = get_data_for_key_metric_and_geographies(profile_key_metric, [geography])
     if indicator_data != None:
         return MetricCalculator.subindicator(indicator_data, profile_key_metric, geography)
     return None
 
-def sibling(profile_key_metric, geography):
-    siblings = geography.get_siblings()
+def sibling(profile_key_metric, geography, versions):
```

Comment:
I don't think this is the right thing to do here. The sibling calculate looks at the % that this geography represents as a function of all the surrounding geographies - e.g. CPT contains 50% of the population in WC